src/index.js

```diff
@@ -1 +1,71 @@
-console.log(6);
\ No newline at end of file
+/* eslint-disable no-unused-vars */
+import DoISuportIt from './style.css';
+
+const api = {
+  mykey: '02e1cd5bf4b503f7aaf9f2e799172b16',
+  base: 'https://api.openweathermap.org/data/2.5/',
+};
+const notificationElement = document.querySelector('.notification');
+const btnEl = document.querySelector('.btn');
+const btnUnitValue = document.querySelector('.btn p .toggle');
+const tempChange = document.querySelector('.btn p .change');
+let temperatureC = 15;
+
+const searchbox = document.querySelector('.search-box');
+/* eslint-disable no-use-before-define */
+searchbox.addEventListener('keypress', setQuery);
+function setQuery(evt) {
+  if (evt.keyCode === 13) {
+    if (searchbox.value !== '') {
+      notificationElement.style.display = 'none';
+      getResults(searchbox.value);
+    } else {
+      notificationElement.style.display = 'block';
+      notificationElement.innerHTML = '<p>City Location cannot be empty</p>';
+      notificationElement.style.display = 'flex';
+    }
+  }
+}
+
```

Comment:
The project requirements specified that you should use ES6 syntax for this project. which means you have to use arrow functions instead of regular functions.